src/Analyzers/Core/Analyzers/FileHeaders/AbstractFileHeaderDiagnosticAnalyzer.cs

```diff
@@ -21,7 +21,7 @@ internal abstract class AbstractFileHeaderDiagnosticAnalyzer : AbstractBuiltInCo
     private static readonly DiagnosticDescriptor s_missingHeaderDescriptor = CreateDescriptorForFileHeader(s_missingHeaderTitle, s_missingHeaderMessage);
 
     private static DiagnosticDescriptor CreateDescriptorForFileHeader(LocalizableString title, LocalizableString message)
-        => CreateDescriptorWithId(IDEDiagnosticIds.FileHeaderMismatch, EnforceOnBuildValues.FileHeaderMismatch, hasAnyCodeStyleOption: true, title, message);
+        => CreateDescriptorWithId(IDEDiagnosticIds.FileHeaderMismatch, EnforceOnBuildValues.FileHeaderMismatch, hasAnyCodeStyleOption: false, title, message);
```

Comment:
>false [](http://example.com/codeflow?start=135&length=5)

I tried this earlier at your suggestion, and I believe it caused me to hit asserts. I assume you don't see those? If not, I can try building this branch locally and see if I can still reproduce.

Reply:
If I'm remembering, the assert was that options were specifies but that this flag was false.

Reply:
> I assume you don't see those?

I haven't tried running this. If CI shows asserts I can review.

Reply:
Found one such issue, now running tests again.